### PR TITLE
Add support for Responsive Layout properties

### LINKED ISSUE #138: AutoLayout properties under the DynamicProperties key should be included in the Yaml output, not .editorstate

### PR DESCRIPTION
**Pull request by lesaltzm**
fixes #138

### CODE REVIEW

File: src/PAModel/ControlTemplates/DynamicProperties.cs [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    // Responsible for handling dynamic properties and their default vaues
+    internal static class DynamicProperties
+    {
+        internal static bool AddsChildDynamicProperties(string template)
+        {
+            return template == "groupContainer";
+        }
+
+
+        private static HashSet<string> _supportsNestedControls = new HashSet<string>()
```
Comment (lesaltzm) [inline src/PAModel/ControlTemplates/DynamicProperties.cs:16]: Most of this will be needed for #142, which I've decided to do as a separate PR

**Review by MikeStall — APPROVED** [review]
